docs/root/start/building/building.rst

```diff
@@ -55,7 +55,7 @@ See `ci/mac_ci_setup.sh` for the specific NDK version used during builds.
 iOS requirements
 ----------------
 
-- Xcode 12.5.1
+- Xcode 13.0
 - Swift 5.x
```

Comment:
Should this be Swift 5.5 considering Xcode 13.0+ is required?

Swift 5.0 was last supported in Xcode 10.2.1 so I'm assume we don't validate that it still compiles with that language version?

Reply:
we might as well just remove this line since they're tied together